lib/action/invoke_api_action.dart

```diff
@@ -117,6 +117,7 @@ class InvokeAPIController {
         errorResponse = response;
       } catch (error) {
         errorResponse = error;
+        throw error.toString();
```

Comment:
don't throw the error - this will stop the app flow. THe intention is for the onApiError to execute and notifies the user.